content/blog/getting-started-with-the-hpe-data-services-cloud-console-public-rest-api.md

```diff
@@ -37,7 +37,9 @@ The website also provides additional information:
 2. The information about the HTTP method, parameters and the responses that are expected from each resource.
 3. The syntax for the HTTP method and path to this resource. Note that this path is a relative path. For the complete path, please add the base-URL documented below.
 4. The body of response is returned in JSON format according to the response status of the REST API.
-   The website also provides the links to download the DSCC OpenAPI definitions in either json or yaml format. Below is an example of the downloaded yaml definition file from the DSCC REST API documentation website.
+
+
+   The website also provides the links to download the cloud console OpenAPI definitions in either json or yaml format. Below is an example of the downloaded yaml definition file from the DSCC REST API documentation website.
```

Comment:
Hi Ron, 

There is the DSCC acronym at the end of this sentence:
...from the DSCC REST API documentation website.


There is also another instance of DSCC acronym in this post section:

**Data Services Cloud Console REST API Details**

Anyone can download this OpenAPI Specification (OAS) v3 definition of the **DSCC** from the following: [Link to the API repository](https://console-us1.data.cloud.hpe.com/doc/api/v1/)